### ISSUE TITLE
Bug with drop

### ISSUE THREAD

**Issue by PwnedCax** [DevilXD/TwitchDropsMiner#387]
Hello , I use twitch drops miner v16.dev. e5f4ea1 for the twitch rivals, but the problem is that it stays stuck on the RIVALS IV GENERIC, while it farms another item on a stream, so it's not really up to date, basically on the software it tells me DROP RUSTORIA SLEEPING BAG, while on twitch it tells me Dhalucard Assault Rifle, and also I left the application running last night but I have the impression that it was not as efficient as before.

**Comment by matarife123** [comment]
The problem I have is that I get the AK drop for example, but the generic drops don't count for me.
![Sin título](https://github.com/DevilXD/TwitchDropsMiner/assets/110930030/c532466c-7e1b-4dd5-8c9b-aea06e3cb630)

**Comment by Leetoi** [comment]
I have the same problem, it laid something, and now it doesn't switch automatically, it just laid a stone, and that's it. And the general drop is frozen.

**Comment by ryanmattt** [comment]
This is a Twitch bug, the Generic drops seem to activate for people after a streamer's specific drop has been claimed. Some suggest watching a steamer with drops enabled that is not part of the event :shrug:.

**Comment by DevilXD** [comment]
There's a couple of things to unpack here:

> on the software it tells me DROP RUSTORIA SLEEPING BAG, while on twitch it tells me Dhalucard Assault Rifle

This isn't really fixable. Twitch has removed the ability to tell which drop is being mined exactly, so the miner has to guess. There's no way to prevent it from guessing incorrectly sometimes. Your best bet is to just ignore what it says, and use the website progress instead. This is especially true now that multiple drops from a single campaign can be progressed at the same time, while the miner updates the progress only one one of them at a time.

> it stays stuck

See #301.

> This is a Twitch bug

Very likely may be, if two campaigns allows for the same stream to be watched to progress both of them, then they both should be progressed at the same time. If that doesn't happen, there's nothing I can do about it.

**Comment by drewgilkey** [comment]
@DevilXD Think that its the "all" channels causing issues. The system doesnt seem to know how to disseminate between what to prioritize. Is there any way to maybe add a quick feature to specify what channels to watch? awesome program btw!

**Comment by DevilXD** [comment]
> its the "all" channels causing issues

Yes, I'm aware.

> maybe add a quick feature to specify what channels to watch?

I think you misunderstood what "quick" means here - adding such a feature couldn't be further from being quick. Also, it goes against the project goals of everything being as automated as possible. If you'd like to learn more, search the already closed issue - this topic was brought up many times in the past already.

If the miner is watching a channel and it's progressing some drops but not others, then either it's a Twitch bug, or an ACL difference, where one campaign has no ACL specified (all channels can progress it), while the other has an ACL, and only specific channels can progress it. ACL channels are given priority over non-ACL ones.

Unless watching the channel "the normal way" progresses the campaigns as expected, without any tracebacks, there isn't much I can do here, unless more specific information on the issue is given.

**Comment by Totesfleisch** [comment]
I just want to throw in my findings on this, i think this isnt really related to the bot here, i've been trying to progress this drop by farming multiple other drops manually, and no channel i tried did progress this campaign, it must be some twitch issue because so far i couldnt progress this event past 1% with neither manual nor bot watching.

**Comment by Totesfleisch** [comment]
I checked some more, and i found that you cannot progress 2 campaigns at the same time anymore, atleast not in the Regards to Rust. I suspected something like this with SMITE when i watched a streamer with time limited drops, that didnt progress the generic drops, but wasnt sure and didnt test since i dont care about the generic ones anyway. However after trying around with Rust, when i go to a channel, who's specific Item i earned already, i will progress the generic drops.
